client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -13,73 +13,102 @@
-      <template v-slot:action="rowData">
-        <div class="float-right">
-          <template v-if="!rowData.edit">
-            <button
-              :aria-label="Translator.trans('item.edit')"
-              class="btn--blank o-link--default"
-              @click="editInstitution(rowData.id)">
-              <i
-                class="fa fa-pencil"
-                aria-hidden="true" />
-            </button>
-          </template>
-          <template v-else>
-            <button
-              :aria-label="Translator.trans('save')"
-              class="btn--blank o-link--default u-mr-0_25"
-              @click="addTagsToInstitution(rowData.id)">
-              <dp-icon
-                icon="check"
-                aria-hidden="true" />
-            </button>
-            <button
-              class="btn--blank o-link--default"
-              :aria-label="Translator.trans('abort')"
-              @click="abortEdit()">
-              <dp-icon
-                icon="xmark"
-                aria-hidden="true" />
-            </button>
-          </template>
-        </div>
-      </template>
-    </dp-data-table>
+
+    <dp-loading
+      class="u-mt"
+      v-if="isLoading" />
+    <template v-else>
+      <dp-column-selector
+        data-cy="institutionList:selectableColumns"
+        :initial-selection="currentSelection"
+        local-storage-key="institutionList"
+        :selectable-columns="selectableColumns"
+        use-local-storage
+        @selection-changed="setCurrentSelection" />
+
+      <dp-data-table
+        ref="dataTable"
+        class="u-mt-2 overflow-x-auto scrollbar-none"
+        data-dp-validate="tagsTable"
+        data-cy="institutionList:dataTable"
+        :header-fields="headerFields"
+        is-resizable
+        :items="institutionList"
+        track-by="id">
+        <template v-slot:name="institution">
+          <ul class="o-list max-w-12">
+            <li>
+              {{ institution.name }}
+            </li>
+            <li class="o-list__item o-hellip--nowrap">
+              {{ date(institution.createdDate) }}
+            </li>
+          </ul>
+        </template>
+        <template
+          v-for="category in institutionTagCategories"
+          v-slot:[category.attributes.name]="institution">
+          <div v-if="!institution.edit">
+            <span>
+              {{ separateByCommas(institution.tags.filter(tag => tag.category.id === category.id)) }}
+            </span>
+          </div>
+          <dp-multiselect
+            v-else
+            v-model="editingInstitutionTags[category.id]"
+            :data-cy="`institutionList:tags${category.attributes.name}`"
+            label="name"
+            multiple
+            :options="getCategoryTags(category.id)"
+            track-by="id" />
+        </template>
+        <template v-slot:action="institution">
+          <div class="float-right">
+            <template v-if="!institution.edit">
+              <button
```

Comment:
You can put the v-if straight on the button. Then we don't need the template tag

Reply:
And I think I would switch the if and else block, that we don't have to negate the if-expression

Reply:
Done with [9845678](https://github.com/demos-europe/demosplan-core/pull/4115/commits/9845678e475ae47fe8ca5fa4e8bd7822d3505762)